### ISSUE TITLE
Train on the entire dataset without CL

### ISSUE THREAD

**Issue by hailuu684** [vlomonaco/core50#7]
Hello Vlomonaco,
I would like to ask if you have ever trained on the entire dataset without CL? I tried to get some baselines but the accuracy is only around 50%. I already used the models which are from simple to complex, for instance, ConvNet, Resnet, and ViT but the accuracies are just around 50%.

**Comment by vlomonaco** [comment]
Hi @hailuu684, this repo is discontinued in favor of: https://avalanche.continualai.org/from-zero-to-hero-tutorial/03_benchmarks

But yes, a couple of baselines for static training are available here: https://arxiv.org/abs/1705.03550

![image](https://github.com/vlomonaco/core50/assets/4135236/715d2f2d-711c-4355-8de0-e93bcde331eb)

**Comment by hailuu684** [comment]
Hi @vlomonaco , this is super helpful for me. I am also interested in CL. Thank you for your reply

**Comment by hailuu684** [comment]
May I ask if have you tried to train with RGB-D images?

**Comment by vlomonaco** [comment]
Yes we did some tests and it was naturally improving performance just treating it as an additional input channel... but I don't think we added it in any paper!

**Comment by hailuu684** [comment]
Hello @vlomonaco , I tried to reproduce the result of mid CNN + FT. Did you experience very high accuracy at the first epoch and then decrease gradually over epochs, finally staying at around 10% of top 1 accuracy, and loss = 2.31? I used vgg18, vit, resnet with pre-trained weights on imagenet, I also change the input size to 128 as you suggested in the paper
